### PR TITLE
[jsk_pcl_ros] Fix flipped negative_ of ExtractIndices (bugfix)

### PR DESCRIPTION
**Pull request by wkentaro**
I mistakenly take it as opposite negative and non negative.
I will send PR to pcl also.

### CODE REVIEW

File: jsk_pcl_ros/include/jsk_pcl_ros/pcl/extract_indices_patch.h [hunk 1]
```diff
@@ -47,11 +47,34 @@ pcl::ExtractIndices<pcl::PCLPointCloud2>::applyFilter (PCLPointCloud2 &output)
...
     output = *input_;
-    for (size_t i = 0; i < indices_->size (); ++i) {
-      for (size_t j = 0; j < output.fields.size(); ++j) {
-        memcpy (&output.data[(*indices_)[i] * output.point_step + output.fields[j].offset],
-                &user_filter_value_, sizeof(float));
-      }
+    if (negative_)
+    {
+      // Prepare the output and copy the data
+      for (size_t i = 0; i < indices_->size (); ++i)
```
Comment (garaemon) [inline jsk_pcl_ros/include/jsk_pcl_ros/pcl/extract_indices_patch.h]: Add `{}` even for has one sentence.

Comment (wkentaro) [inline jsk_pcl_ros/include/jsk_pcl_ros/pcl/extract_indices_patch.h]: I'd like to keep this file same as the one in pcl. Would you mind that?
In pcl library the above seems the common style.

Comment (wkentaro) [inline jsk_pcl_ros/include/jsk_pcl_ros/pcl/extract_indices_patch.h]: I'm trying keep same code as https://github.com/PointCloudLibrary/pcl/blob/master/filters/src/extract_indices.cpp.

Comment (garaemon) [inline jsk_pcl_ros/include/jsk_pcl_ros/pcl/extract_indices_patch.h]: I see.

**Pull request merged**